charts/pravega/README.md

```diff
@@ -42,13 +42,7 @@ The following table lists the configurable parameters of the Pravega chart and t
 | `zookeeperUri` | Zookeeper service address | `zk-client:2181` |
 | `externalAccess.enabled` | Enable Pravega external access | `false` |
 | `externalAccess.type` | Pravega external access type | `LoadBalancer` |
-| `bookkeeper.image.repository` | Image repo for Bookkeeper image | `pravega/bookkeeper` |
-| `bookkeeper.replicas` | Replicas for Bookkeeper | `3` |
-| `bookkeeper.storage.className` | Storage class for Bookkeeper volumes | `standard` |
-| `bookkeeper.storage.ledgerVolumeRequest` | Request storage for ledgerVolume | `10Gi` |
-| `bookkeeper.storage.journalVolumeRequest` | Request storage for journalVolume | `10Gi` |
-| `bookkeeper.storage.indexVolumeRequest` | Request storage for indexVolume | `10Gi` |
-| `bookkeeper.autoRecovery`| Enable Bookkeeper autoRecovery | `true` |
+| `bookkeeperUri` | URL for Bookkeeper Servers | `bookkeeper-client:3181` |
```

Comment:
Also, the documentation needs to point to the new Bookkeeper operator as a pre-requisite prior to deploy this Operator and Pravega, right?

Reply:
Sure. Will add this. While we can't mandate that we can recommend it.